Resources/Prototypes/Catalog/Fills/Lockers/service.yml

```
@@ -32,111 +40,153 @@
   suffix: Filled
   parent: ClosetChef
   components:
-  - type: StorageFill
-    contents:
-      - id: VariantCubeBox
-      - id: BoxMousetrap
-        amount: 2
-      - id: SprayBottleWater
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:NestedSelector
+        tableId: FillClosetChef
+
+- type: entityTable
+  id: FillClosetChef
+  table: !type:AllSelector
+    children:
+    - id: VariantCubeBox
+    - id: SprayBottleWater
+    - id: FoodBoxCloth
+    - id: ReagentContainerSugar
+    - id: FoodCondimentBottleEnzyme
+    - id: DrinkSoyMilkCarton
+    - !type:AllSelector
+      rolls: !type:ConstantNumberSelector
+        value: 2
+      children:
+      - id: DrinkMilkCarton
+      # really, milk should go in the fridge. Unfortunately saltern only has freezers.
+      # yes, I'm using this as an excuse to not have to do extra work.
       - id: ReagentContainerFlour
-        amount: 2
-      - id: FoodBoxCloth
-        amount: 1
-      - id: ReagentContainerSugar
+      - id: BoxMousetrap
+    - !type:AllSelector
+      rolls: !type:ConstantNumberSelector
+        value: 3
+      children:
       - id: FoodCondimentPacketSalt
-        amount: 3
       - id: FoodCondimentPacketPepper
-        amount: 3
-      - id: FoodCondimentBottleEnzyme
-      # really, milk should go in the fridge. Unfortunately saltern only has freezers.
-      # yes, I'm using this as an excuse to not have to do extra work.
-      - id: DrinkMilkCarton
-        amount: 2
-      - id: DrinkSoyMilkCarton
 
 - type: entity
   id: ClosetJanitorFilled
   suffix: Filled
   parent: ClosetJanitor
   components:
-  - type: StorageFill
-    contents:
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:NestedSelector
+        tableId: FillClosetJanitor
+
+- type: entityTable
+  id: FillClosetJanitor
+  table: !type:AllSelector
+    children:
+    - id: WetFloorSign
+      amount: !type:ConstantNumberSelector
+        value: 3
+    - id: LightReplacer
+    - id: BoxLightMixed
+    - id: Holoprojector
+    - !type:AllSelector
+      rolls: !type:ConstantNumberSelector
+        value: 2
```

Same here, you can just do
```suggestion
    - !type:AllSelector
      rolls: 2
```